libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -240,23 +238,22 @@ class GrayboxPromote extends LitElement {
     return html`
       <div class="wrapper">
         ${this.getSpTokenTask.render({
-          pending: () => html`<p>Logging in to sharepoint...</p>`,
-          complete: () => {},
-          error: (err) =>
-            html`<p>Error getting sharepoint token: ${err.message}</p>`,
-        })}
+    pending: () => html`<p>Logging in to sharepoint...</p>`,
+    complete: () => {},
+    error: (err) => html`<p>Error getting sharepoint token: ${err.message}</p>`,
+  })}
         ${this.promoteTask.render({
-          pending: () => html`<p>Promoting...</p>`,
-          complete: (i) => i,
-          error: (err) => html`<p>${err}</p>`,
-        })}
+    pending: () => html`<p>Promoting...</p>`,
+    complete: (i) => i,
+    error: (err) => html`<p>${err}</p>`,
+  })}
         ${
-          [1,2].includes(this.promoteTask.status) ? '' :
-          this.getValuesTask.render({
-          pending: () => html`<p>Loading...</p>`,
-          complete: (i) => i,
-          error: (err) => html`<p>${err.message}</p>`,
-        })}
+  [1, 2].includes(this.promoteTask.status) ? ''
+    : this.getValuesTask.render({
+      pending: () => html`<p>Loading...</p>`,
+      complete: (i) => i,
+      error: (err) => html`<p>${err.message}</p>`,
+    })}
```

Comment:
Indentation looks funky in this section.

Reply:
Same as above, the eslint formatter is changing it this way. Any other indentation I have tried is throwing an eslint error.